evalml/pipelines/components/estimators/regressors/arima_regressor.py

```diff
@@ -26,6 +26,8 @@ class ARIMARegressor(Estimator):
         max_d (int): Maximum Differencing degree. Defaults to 2.
         max_q (int): Maximum Moving Average order. Defaults to 5.
         seasonal (boolean): Whether to fit a seasonal model to ARIMA. Defaults to True.
+        sp (int): Period for seasonal differencing, specifically the number of periods in each season. If "detect" and the time series is one of
+                  a few standard frequencies, this model will automatically detect this parameter, defaulting to 1 (no seasonality). Defaults to "detect".
```

Comment:
I find `....defaulting to 1 (no seasonality). Defaults to "detect".` a little confusing. Maybe something along the liens of: 

`If "detect", this model will automatically detect this parameter (given the time series is a standard frequency) and will fall back to 1 (no seasonality) if it cannot be detected. Defaults to "detect".`